### PR TITLE
Making YamlWriter a public class

### PR DESCRIPTION
**Pull request by tomjschneider**
Making `YamlWriter` a public class so it can be utilized by other projects looking to conform to the Power App specific flavor of YAML.

### CODE REVIEW

**Review by NavneetThekkumpat — APPROVED** [review]

File: src/PAModel/PAConvert/Yaml/YamlWriter.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@ namespace Microsoft.PowerPlatform.Formulas.Tools.Yaml
     /// Notably, property values are always either multi-line escaped or
     /// prefixed with '=' to block yaml from treating it as a yaml expression.
     /// </summary>
-    internal class YamlWriter
+    public class YamlWriter
     {
```
Comment (lesaltzm) [inline src/PAModel/PAConvert/Yaml/YamlWriter.cs:16]: Double checking, is there anything in here that becomes public that we don’t need to be?

Comment (tomjschneider) [reply src/PAModel/PAConvert/Yaml/YamlWriter.cs:16]: The class itself isn't tied to anything else and everything else in the same namespace is marked `internal`. I want everything that is already public on it to be available, so I think we're all good here.

Comment (tomjschneider) [reply src/PAModel/PAConvert/Yaml/YamlWriter.cs:16]: Ah you know what - `NormalizeNewlines` is only used within this class. Let me mark that as private, it doesn't need to be exposed.

Comment (tomjschneider) [reply src/PAModel/PAConvert/Yaml/YamlWriter.cs:15]: Okay, changed that to private and removed the static designation as it didn't seem necessary.

**Review by lesaltzm — APPROVED** [review]

**Pull request merged**